sensor_msgs/CameraInfo  camera_info
float64                 z_offset
float64                 z_scaling
